src/components/nav.html

```diff
@@ -0,0 +1,33 @@
+<div class="header_main_navigation">
+  <a href="" class="header_main_navigation_logo">HOFMANN</a>
+  <div class="menu">
+    <nav>
+      <ul class="menu_list">
+        <include src="./components/nav-list.html" locals='{
+            "classList":"menu_list_item",
+            "list": "[`MISSION`, `SERVICES`, `ABOUT`]"
+          }'></include>
+      </ul>
+    </nav>
+    <div class="menu_action">
+      <a class="button_mobile" id="toggle">
+        <img src="../assets/icons/hamburger-menu.svg" alt="Search" class="hamburger_menu_mobile" />
+      </a>
+      <button>TAKE A TOUR</button>
+    </div>
+
+  </div>
+</div>
+<script>
```

Comment:
Feedback: Actually we don't need to use JS in this project! Just static website base on html.css.